src/app/movies/movies.component.html

```diff
@@ -15,10 +15,10 @@
   </div>
 </div>
 <div class="movie" *ngFor="let movie of (movies$ | async)">
-  {{movie.type}} - {{movie.id}}
+  {{movie.type}} {{movie.id}} {{movie.loading}}
```

Comment:
no jak chcesz mieć napisane zamiat samego "true" "loading: true", to co tu trzeba dopisać? :p